Adding Padding after Footer.

avalynndev opened the pull request:
In Chromebook, and laptops the footer doesn't have space at the bottom, making it stick to the bottom of the website. which makes it look wierd.
![image](https://github.com/heysagnik/Linkees/assets/100826194/ae43347b-9d0c-4c60-9c49-e7b764ae256c)
so i thought of adding a little place

also i didnt check if u installed tailwind or no, in a hurry. but u get the idea..

heysagnik commented:
Nope this project doesn't use `tailwind-css`.

You can use ```padding-bottom: 2.5rem; ``` instead of that class in the css file.

avalynndev commented:
I removed `tailwind-css` code and added it in the `components.css` file, it works for me when I opened the Dev tools and updating CSS. (`Ctrl` + `Shift` + `I`). 
Anything else to update lemme know
